docs/developer-docs/flags/remember.md

```diff
@@ -0,0 +1,186 @@
+|  2 | Remember  | Mem   | 16-bit | 0x1234 |     |       |        |         |       |
+
+In this example, the `Delta` value of `0x1234`, the value from the previous frame, is subtracted from the `Mem` value of `0x1234` and Remembered.
+
+### Simple Example - With Operators
+
+| ID | Flag      | Type  | Size   | Memory | Cmp | Type  | Size   | Mem/Val | Hits  |
+| -- | --------- | ----- | ------ | ------ | --- | ----- | ------ | ------- | ----- |
+|  1 | Remember  | Mem   | 16-bit | 0x1234 |  -  | Delta | 16-Bit | 0x1224  |       |
+
+The Remember condition can optionally take two operand and process a value based on an operator such as multiplication, division, addition, subtraction, and others. In this example, we used the subtraction operator to store the equivalent value as in the first example, but we do it in a single line.
+
+### Using the Stored Value
+
+To use the stored value, specify the `Recall` operator as the Type of operand:
+
+| ID | Flag      | Type   | Size   | Memory | Cmp | Type  | Size   | Mem/Val | Hits  |
+| -- | --------- | ------ | ------ | ------ | --- | ----- | ------ | ------- | ----- |
+|  2 |           | Recall |        |        |  =  | Value |        | 0x05    |       |
+
+In this example it uses the stored value and compares it to a constant value of 5. However, using the stored value once does not show off the main usefulness of `Remember`. 
+
+| ID | Flag      | Type   | Size   | Memory | Cmp | Type  | Size   | Mem/Val | Hits  |
+| -- | --------- | ------ | ------ | ------ | --- | ----- | ------ | ------- | ----- |
+|  1 | Remember  | Mem    | 16-bit | 0x1234 |  -  | Delta | 16-Bit | 0x1224  |       |
+|  2 | And Next  | Recall |        |        |  =  | Value |        | 0x05    |       |
+|  3 | Add Hits  | Mem    | 8-Bit  | 0x1000 |  =  | Value |        | 0x03    | 1 (0) |
+|  4 | And Next  | Recall |        |        |  =  | Value |        | 0x14    |       |
+|  5 | Add Hits  | Mem    | 8-Bit  | 0x1000 |  =  | Value |        | 0x03    | 1 (0) |
+|  6 | And Next  | Recall |        |        |  =  | Value |        | 0x14    |       |
+|  7 | Add Hits  | Mem    | 8-Bit  | 0x1000 |  =  | Value |        | 0x03    | 1 (0) |
+|  8 | Measured  | Value  |        | 0x00   |  =  | Value |        | 0x00    | 3 (0) |
+
+In this example, we are using the stored value in multiple chains within an Add Hits chain without having to recalculate the value each time. Here we are looking for specific increases of `0x1234` to occur while the value of `0x1000` is equal to 3.  Perhaps `0x1234` is money, `0x1000` is the current level, and you want to make sure the player collects a gem worth 5 units of money, one worth 20, and one worth 100 in level 3. `Recall` is used here to check all these values of the difference between Mem and Delta without having to recalculate it each time. See the [`Add Hits`](/developer-docs/flags/addhits-and-subhits) document for more information about how that condition works.
```

Comment:
```suggestion
In this example, we are using the stored value in multiple chains within an Add Hits chain without having to recalculate the value each time. Here we are looking for specific increases of `0x1234` to occur while the value of `0x1000` is equal to 3.  Perhaps `0x1234` is money, `0x1000` is the current level, and you want to make sure the player collects a gem worth 5 units of money, one worth 20, and one worth 100 in level 3. `Recall` is used here to check all these values of the difference between Mem and Delta without having to recalculate it each time. See the [`Add Hits`](/developer-docs/flags/addhits-subhits) document for more information about how that condition works.
```

Reply:
Caught that it doesn't have -flag, missed that it doesn't have -and

Fixed